src/tablib/formats/_csv.py

```diff
@@ -42,8 +42,9 @@ def import_set(cls, dset, in_stream, headers=True, **kwargs):
 
         rows = csv.reader(in_stream, **kwargs)
         for i, row in enumerate(rows):
-
-            if (i == 0) and (headers):
+            if i < skip_lines:
+                continue
+            if (i == skip_lines) and (headers):
```

Comment:
Remove redundant parentheses, like in `src/tablib/formats/_xls.py`:

```suggestion
            if i == skip_lines and headers:
```